### DISCUSSION TITLE
Contact with Vesoft sales department

### DISCUSSION THREAD

**Discussion by Thoughtful1** [vesoft-inc/nebula#5985] [General]
for some reason Vesoft company aren't responding to mails with questions about Enterprise edition of Nebula Graph. Does anyone know how may I get in touch with Vesoft company?

**Comment by wey-gu** [comment]
Sorry @Thoughtful1 

There could be something wrong on the sale pipeline, pinged our sales team already🫡.

In the meantime, could you drop mail to me with some context so I could also directly fwd to the team?

--
wey.gu at vesoft dot com

Thanks!

**Reply by Thoughtful1** [reply]
I just sent you @wey-gu an email on weyl.gu@gmail.com

**Reply by wey-gu** [reply]
Got it, fwd to the sales team already :-D thanks!
